models/public/retinaface-anti-covid/model.yml

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+description: >-
+  RetinaFace-Anti-Cov is a customized one stage face detector to help people protect
+  themselves from CovID-19. More details provided in the paper <https://arxiv.org/abs/1905.00641>
+  and repository <https://github.com/deepinsight/insightface/tree/master/RetinaFaceAntiCov>
+task_type: detection
```

Comment:
@IRDonch is there any problem with multiple `task_type` for our tools? Since we have multi-purpose models, we should support them in this tag, shouldn't we?

Reply:
We've already discussed it internally, but for the record, I'll restate my position here: I don't object to this in principle, but the `task_type` field is part of Info Dumper's output, so it has to remain as-is. It would be possible to introduce an `additional_task_types` field containing a list of, well, additional task types.